prijateli_tree/app/locales/en.yml

```diff
@@ -0,0 +1,112 @@
+en:
+  homepage:
+    login: "Log In"
+    facilitators: "Only for facilitators!"
+    school: "School"
+    student_name: "First and Last Name"
+    student_language: "Language of Student"
+    facilitator_username: "Facilitator Username"
+    facilitator_password: "Facilitator Password"
+
+  ready_to_play:
+    get_ready: "Get ready to play Red or Blue!"
+    im_ready: "Im ready!"
+
+  waiting_screen:
+    waiting: "Waiting for others..."
+
+  rules_screen:
+    how_to_play: "How to Play?"
+    instructions: "You can replay the video as many times as you want. Not knowing the rules lowers your chances of winning. Ask your teacher if you have any questions about the game."
+    i_understand: "I understand the rules!"
+    next: "Next"
+
+  quiz_colors:
+    quick_quiz: "Quick Quiz!"
+    question: "Question"
+    instructions: "To win, you have to pick the bag the computer chose. What colors are the bags the computer chooses?"
+    red_and_blue: "Red and Blue"
+    green_and_yellow: "Green and Yellow"
+    black_and_white: "Black and White"
+
+  quiz_balls:
+    colors_in_bag: "What colors are the balls in the blue bag?"
+    four_blue_two_red: "4 blue and 2 red"
+    three_blue_four_red: "3 blue and 4 red"
+    five_blue_two_red: "5 blue and 2 red"
+
+  quiz_peers:
+    what_will_you_observe: "What will you observe to help you learn which bag the computer chose?"
+    one_ball_few_peers: "One ball and a few peers' guesses"
+    two_balls: "Two balls"
+    one_ball_all_peers: "One ball and all peers' guesses"
+
+  intro_practice:
+    now_you_will_play: "Now you will play 2 practice games to get used to the rules."
+    since_these_rules: "Since these are practice rounds, we will not reward anyone for their performance."
+
+  practice_game_video:
+    practice_game: "Practice Game"
+    explanation: "Explanation"
+    score: "Score"
+    feel_free: "Feel free to watch the video again to remind yourself of the rules."
+
+  practice_game_first_round:
+    round: "Round"
+    the_computer_chose: "The computer chose a bag and split the balls randomly to each player."
+    your_ball_is_red: "Your ball is RED"
+    your_ball_is_blue: "Your ball is BLUE"
+    guess_color: "Guess the color of the bag:"
+    red: "RED"
+    blue: "BLUE"
+
+  practice_game_other_rounds:
+    see_what_others_guessed: "See what some other players guessed in the last round:"
+    code_guessed_red: "Code guessed RED."
+    code_guessed_blue: "Code guessed BLUE."
+    name_guessed_red: "Name guessed RED."
+    name_guessed_blue: "Name guessed BLUE."
+    your_guess_was_red: "Your guess in the last round was RED."
+    your_guess_was_blue: "Your guess in the last round was BLUE."
+
+  practice_game_results:
+    results: "Results"
+    you_won: "You won!!!"
```

Comment:
Let's keep the punctuation to one as opposed to many.

Reply:
Doneee